WreckEngine/TextEntity.cpp

```diff
@@ -1,6 +1,17 @@
 #include "TextEntity.h"
 
 TextEntity::TextEntity(const std::string _message, const std::string& _font, Text::Justify _vertical, Text::Justify _horizontal, const uint32_t height, const uint32_t width)
-	: Entity(), font(Text::loadWinFont(_font, height, width)), message(_message), vertical(_vertical), horizontal(_horizontal) {}
+    : Entity()
+    , vertical(_vertical)
+    , horizontal(_horizontal)
+    , inst(make_shared<Text::Instance>())
```

Comment:
the general idea here is that if the exact same text is reused between different entitys, it could be shared right? would that work, given that offset and alignment might change between entitys?